src/library_webgpu.js

```diff
@@ -1639,7 +1651,20 @@ var LibraryWebGPU = {
     }
 
     function makeRenderPassDescriptor(descriptor) {
-      {{{ gpu.makeCheckDescriptor('descriptor') }}}
+      {{{ gpu.makeCheck('descriptor') }}}
+      var nextInChainPtr = {{{ makeGetValue('descriptor', C_STRUCTS.WGPURenderPassDescriptor.nextInChain, '*') }}};
+      
+      var maxDrawCount = undefined;
+      if (nextInChainPtr !== 0) {
+        var sType = {{{ gpu.makeGetU32('nextInChainPtr', C_STRUCTS.WGPUChainedStruct.sType) }}};
+#if ASSERTIONS
+        assert(sType === {{{ gpu.SType.RenderPassDescriptorMaxDrawCount }}});
+#endif
+        var renderPassDescriptorMaxDrawCount = nextInChainPtr;
+        {{{ gpu.makeCheckDescriptor('renderPassDescriptorMaxDrawCount') }}}
+        maxDrawCount = {{{ gpu.makeGetU64('renderPassDescriptorMaxDrawCount', C_STRUCTS.WGPURenderPassDescriptorMaxDrawCount.maxDrawCount) }}};
+      }
```

Comment:
nit: assert that `descriptor.nextInChain->next` is 0, or implement the loop over the chain

Reply:
`nextInChain.chain`?

```
"WGPURenderPassDescriptorMaxDrawCount": [
                "chain",
                "maxDrawCount"
            ],
```

Reply:
yes, sorry

Reply:
actually it's the same thing, because we're really doing a reinterpret cast in the middle from WGPUChainedStruct to WGPURenderPassDescriptorMaxDrawCount.

```c
typedef struct WGPURenderPassDescriptor {
    WGPUChainedStruct const * nextInChain;
    // ...
} WGPURenderPassDescriptor;

typedef struct WGPUChainedStruct {
    struct WGPUChainedStruct const * next;
    WGPUSType sType;
} WGPUChainedStruct;
```